docs/gitbook/examples/to-do-list.md

```diff
@@ -1,425 +1,259 @@
 
-## To Do List
+# To-do List Example Application
 
-<div class="tips danger">
-  <p><span></span>TODO</p>
-  <p>This example needs a shiny new UI</p>
-</div>
+In the `packages/app-todo-list` directory, we provide a simple To-do List application running on an Interbit blockchain. The application can add items to the to-do list, edit existing items, and toggle the completed state of items.
 
-
-In the `InterbitSDK/examples/to-do-list` directory, we have provided a simple To Do List application running on an Interbit blockchain. The application can add items that need to be done to the list, however those items cannot be toggled.
-
-The following is a brief description of the example To Do application as it currently exists, followed by a tutorial on how to add functionality to toggle To Do items.
+This application was created using the Interbit [Template app](./template), and what follows is a description of the app and how to recreate it from the template. The To-do List uses the same Interbit configuration as the template. As such, to-do items are added to the user's private chain.
```

Comment:
Whenever we use just "Template" to refer to the Template application, the capitalization should be consistent. I don't have a strong preference, although I'd be inclined to capitalize, i.e. "Template", but lowercase works too.

Reply:
I also prefer the capitalized version here as it's an app name. It's also helpful to differentiate from templates in general.